bin/dispatch-conf

```diff
@@ -115,8 +115,13 @@ class dispatch:
         count = 0
 
         perform_conf_update_session_hooks("pre-session")
-
-        config_root = portage.settings["EPREFIX"] or os.sep
+        
+        config_root = (
+            portage.settings["EROOT"]
+            or portage.settings["ROOT"]
+            or portage.settings["EPREFIX"]
+            or os.sep
+        )
```

Comment:
Since portage.settings["EROOT"] is always non-empty, we may as well just do this:
```
config_root = portage.settings["EROOT"] 
```

Reply:
updated!